tests/dummy/app/router.js

```diff
@@ -7,6 +7,14 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
+  this.route('simple-chart-donut');
```

Comment:
I think the routes are a little bit more understandable if they are just `donut` and not `simple-chart-donut`.